docs/root/api-v1/route_config/route.rst

```diff
@@ -338,7 +338,8 @@ Headers
   {
     "name": "...",
     "value": "...",
-    "regex": "..."
+    "regex": "...",
```

Comment:
Do we need this in the v1 API? If you don't have a specific need for this, we prefer to feature freeze v1 and not add new things.

Reply:
We are using the v1 APIs internally and it will be some time before we switch to v2.

Reply:
OK, that's fine for now. We are likely to start pushing harder on this over time so we can turn down v1 at some point, but we haven't announced anything yet and there will be at least a quarter or two notice.